Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -254,4 +261,10 @@ stages:
             dependsOn: GetSkillsByLabel
           - name: GetJobProfilesSummary
             azureFunctionName: '${{ variables.WebAppPrefix }}-stax-jpsummary-fa'
-            dependsOn: GetSkillById
\ No newline at end of file
+            dependsOn: GetSkillById
+          - name: GetSkillsById
```

Comment:
Is this correct?  We're not deploying GetSkillsById (plural) to any other environment.  GetSkillById (singular) is already defined for this environment

Reply:
I think when you look at the full file it might be okay:

`- name: GetOccupationsByLabel
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-occbylabel-fa'
            # This property makes the generated deployment job for this function depend on the 'GetAllSkills' deployment job generated by it's entry in previous array element
            # This causes the generated deployment job to delay execution until the 'GetAllSkills' deployment job has finished.
            # As you add items to this array, please reference them back to the task in the same place in the previous
            # array element, so it gives us a partitioned deployment window!
            dependsOn: GetAllSkills
          - name: GetOccupationsWithMatchingSkills
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-occsmatchskills-fa'
            dependsOn: GetAllOccupations
          - name: GetSkillsGapForOccupationAndGivenSkills
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-sgoccsgskills-fa'
            dependsOn: GetAllSkillsForOccupation
        -
          - name: GetSkillsByLabel
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-skillsbylabel-fa'
            dependsOn: GetOccupationsByLabel
          - name: GetSkillById
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-skillbyid-fa'
            dependsOn: GetOccupationsWithMatchingSkills
          - name: GetJobProfileByTitle
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-jpbytitle-fa'
            dependsOn: GetSkillsGapForOccupationAndGivenSkills
        -
          - name: GetJobProfilesBySearchTerm
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-jpbyterm-fa'
            dependsOn: GetSkillsByLabel
          - name: GetJobProfilesSummary
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-jpsummary-fa'
            dependsOn: GetSkillById
          - name: GetSkillsById
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-skillsbyid-fa'
            dependsOn: GetOccupationsWithMatchingSkills
          - name: GetOccupationsWithMatchingSkillsDetailed
            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-occmsd-fa'
            dependsOn: GetJobProfileByTitle`

Reply:
Line 264 sets a dependency for the deployment that's defined on line 262 - 264
Line 265 defines a new deployment, that deployment (GetSkillsById with an s at the end of Skills) doesn't appear anywhere else in the pipeline

Reply:
GetSkillsById was renamed to GetSkillById, a merge conflict may have created a duplicate entry.  Removed GetSkillsById from PP which now aligns with SIT